### PR TITLE
Use Docsy variable name for blue

### PR DESCRIPTION
**Pull request by sftim**
As blue is our site's primary color, use the Docsy variable for that: `$primary`.

Here's a [preview](https://deploy-preview-48229--kubernetes-io-main-staging.netlify.app/)

Helps with issue https://github.com/kubernetes/website/issues/41171

/area web-development

### CODE REVIEW

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -742,18 +742,18 @@ body.glossary {
...
-        background-color: $blue;
+        background-color: $primary;
       }
       // darken on hover
       .canonical-tag:hover {
         background: darken(#999999, 15%)
       }
       .canonical-tag.active-tag:hover {
-        background: darken($blue, 15%)
+        background: darken($primary, 10%)
```
Comment (sftim) [inline assets/scss/_custom.scss:752]: Very slight tweak here to how the glossary displays.

File: assets/scss/_skin.scss [hunk 2]
```diff
@@ -1,5 +1,7 @@
-$blue: #326ce5;
+$primary: #326ce5;
+$secondary: #495057;
+
 $light-grey: #f7f7f7;
-$dark-grey: #303030;
 $medium-grey: #4c4c4c;
+$dark-grey: #303030;
```
Comment (sftim) [inline assets/scss/_skin.scss:6]: ordered: light to dark

**Review by dipesh-rawat — COMMENTED** [review]
This change makes sense and is a nice improvement.
Thanks!

/lgtm

**Review by Arhell — COMMENTED** [review]
lgtm

**Comment by sftim** [comment]
Rebased after merge of #48194

**Review by dipesh-rawat — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/pull/48362

**Comment by sftim** [comment]
@katcosgrove, are you happy to approve this? AIUI, SCSS changes need tech lead or chair approval.

**Comment by katcosgrove** [comment]
Reasonable change. Thanks!

/approve

**Pull request merged**